utils/utils.tsx

```diff
@@ -1866,6 +1867,28 @@ export function getRoleFromTrackFlow() {
     return {started_by_role: startedByRole};
 }
 
+const TrackFlowSources: Record<string, string> = {
+    wd: 'webapp-desktop',
+    wm: 'webapp-mobile',
+    m: 'mobile-app',
+    d: 'desktop-app',
+};
+
+function getTrackFlowSource() {
+    if (isMobileApp()) {
```

Comment:
The mobile app is a separate repo and has it's own views, would this always be false?

Reply:
My goal for this method was to be reusable even for other events. Do we ever include a webview in the mobile app? If not, it will always be false so it can be removed.

Reply:
I'm not 100% sure if we ever include a webview in our mobile app, you would have to confirm with someone on the mobile team. I think it's unlikely though

Reply:
Ah ok, I will try to verify this and if there is no clear outcome, I will remove it anyway. We can revisit this in the future if there is a case. Thanks!

Reply:
It looks like the only WebView used is for SSO, and from what I can understand, it's for the redirect back. So we can safely remove the mobile app check for now. Thanks for the catch!